src/AudioUnit/AudioUnit.cs

```diff
@@ -644,7 +644,7 @@ public unsafe AudioUnitParameterInfo[] GetParameterList (AudioUnitScopeType scop
 			}
 
 			var info = new AudioUnitParameterInfo [data.Length];
-			size = (uint) Marshal.SizeOf (typeof (AudioUnitParameterInfoNative));
+			size = (uint)sizeof (AudioUnitParameterInfoNative);
```

Comment:
This is internal, so I'm not sure how to test that.

Reply:
But the definition definitely should return the same values:
https://github.com/xamarin/xamarin-macios/blob/b8f9dccdcc09f43a44d591fd0eaf9ba954ea2752/src/AudioUnit/AudioUnit.cs#L235